.github/workflows/ci.yaml

```diff
@@ -86,7 +86,7 @@ jobs:
       tests_glob: ${{ steps.info.outputs.tests_glob }}
       tests: ${{ steps.info.outputs.tests }}
       skip_coverage: ${{ steps.info.outputs.skip_coverage }}
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-22.10
```

Comment:
Not sure `22.10` is available.
https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories

Reply:
Yeah its not going to work

https://github.com/actions/runner-images#available-images shows that ubuntu-22.04 is deploying, but I'm not sure what that means.

I think we have to wait for microsoft to fix their mirror

Reply:
https://github.com/actions/runner-images/issues/9733